### PR TITLE
Add batch scripts to start/stop zeppelin on windows

### PR DESCRIPTION
**Pull request by derlin**
Add the following Windows scripts (and update README):

* `env.cmd` (to set variables used by the other two)
* `start-zeppelin.cmd`
* `stop-zeppelin.cmd`

They have been tested on a Windows 10 Professional Edition and should (in theory) work on any Windows distribution.

_Note_: I chose to create separate files so that users can run scripts by double-clicking on them in the explorer. It spares them the hassle of opening a terminal.

TODO:

* also provide a `refresh.cmd` script

### CODE REVIEW

File: README.md
Comment (vgazzola) [inline README.md]: typo on extention: s/cdm/cmd/

File: start-zeppelin.cmd
Comment (vgazzola) [inline start-zeppelin.cmd]: no need to scream :)

Comment (derlin) [reply start-zeppelin.cmd]: oups, sorry. Kind of tired :).

File: README.md
Comment (a-pagano) [inline README.md]: Actually this is not correct. You need to specify the memory in Gb as an int (see `env.sh`: `export ZEPPELIN_INTP_MEMORY=${ZEPPELIN_INTP_MEMORY:--Xmx${INTEPRETER_MEMORY}g}`). The "-Xmx" and "g" prefix and suffix will be added by the script :)

File: README.md
Comment (a-pagano) [inline README.md]: I'm nitpicking here but I would maybe add "On Linux **and macOS**"

File: stop-zeppelin.cmd
Comment (a-pagano) [inline stop-zeppelin.cmd]: This also needs to be lowercase :)

File: README.md
Comment (vgazzola) [reply README.md]: maybe `Unix-like`?

File: README.md
Comment (derlin) [reply README.md]: From what I recall, 

    export ZEPPELIN_INTP_MEMORY=${ZEPPELIN_INTP_MEMORY:--Xmx${INTEPRETER_MEMORY}g}

means either `ZEPPELIN_INTP_MEMORY` is already defined (and thus does not change), or it is set to `-Xmx${INTEPRETER_MEMORY}g`. So if we override it, the `-Xmx` is never added by the script ! 

Just to be sure, I ran:
```
export ZEPPELIN_INTP_MEMORY=4
./zeppelin.sh --start
echo $ZEPPELIN_INTP_MEMORY # gives 4
```
Zeppelin starts but I cannot run any cell because it passes `4` as startup option.

Comment (a-pagano) [reply README.md]: Ah! You are completely correct my bad! What do you think about changing it to
```
export ZEPPELIN_INTP_MEMORY="-Xmx${ZEPPELIN_INTP_MEMORY:-${INTEPRETER_MEMORY}}g"
```
 

This way the user never has to care about the Xmx flag

Comment (derlin) [reply README.md]: I like the idea, but I would suggest using another variable to avoid the case of the script running twice in the same shell... we would end up with `-Xmx-Xmx4gg` :(.

Comment (a-pagano) [reply README.md]: Good catch!

**Comment by a-pagano** [comment]
Hi @derlin. Any progress on this issue? I can take care of amending your PR according to the reviews you received if you do not have the time to do it yourself. You just need to authorize pushes by maintainers to your branch (https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/). This way we can merge your code

**Comment by derlin** [comment]
Sorry, not completely at ease with how pull requests work. Added new commits following your comments.

**Review by vgazzola — APPROVED** [review]
Looks good to me, @a-pagano I'll let you have a look

**Review by a-pagano — APPROVED** [review]
I've taken the liberty to fix a minor mistake for the export of the ZEPPELIN_INTP_MEMORY variable other than that looks good to me!

**Comment by a-pagano** [comment]
@derlin: Thanks for your contribution 👌

**Pull request merged**
